cookbooks/aws-parallelcluster-slurm/recipes/config/config_slurm_accounting.rb

```diff
@@ -24,6 +24,23 @@
   action :create_if_missing
 end
 
+template "#{node['cluster']['slurm']['install_dir']}/etc/slurm_external_slurmdbd.conf" do
+  source 'slurm/external_slurmdbd/slurm_external_slurmdbd.conf.erb'
+  owner "#{node['cluster']['slurm']['user']}"
+  group "#{node['cluster']['slurm']['group']}"
+  mode '0600'
+  action :create_if_missing
+  variables(
+    dbd_host: "localhost",
```

Comment:
Nitpick: If we are never overriding the dbd_host and storage_port and are always hardcoded to localhost and 3306 respectively, then they should be hardcoded in template itself.